src/main/java/io/codelex/Exceptions/practice/Exercise2.java

```diff
@@ -0,0 +1,40 @@
+package io.codelex.Exceptions.practice;
+
+import javax.swing.plaf.synth.SynthOptionPaneUI;
+import java.util.Scanner;
+
+public class Exercise2 {
+    public static void main(String[] args) {
+
+        methodA();
+
+    }
+
+    public static void methodA() throws ArithmeticException {
+        try {
+            methodB();
+        } catch (ArithmeticException e) {
```

Comment:
`After each try{} block put a catch{} block which catches the exception, prints a stack trace, and throws the exception object to its caller`

Reply:
Ieliku, lai met kļūdu tam saucējam